components/Capping/Capping.jsx

```diff
@@ -0,0 +1,37 @@
+/* @flow */
+import styles from './styles.scss'
+import React from 'react'
+import classnames from 'classnames'
+
+type Props = {
+  cap: "top" | "bottom" | "left" | "right"| null,
+  className:string,
+  children ?: Array<React.Node>,
+
+}
+
+export default class Capping extends React.Component<Props> {
+  static defaultProps = {
+
+  }
+  props: Props
+  render() {
+
+    const {
+      children,
+      className,
+      cap,
+    } = this.props
+
+    const css = [
+      className,
+      styles.cap,
+      styles[`cap-${cap}`],
+    ]
+     return (
+     <div>
+       <button className={classnames(css)}>{children}</button>
```

Comment:
This should use the button component right @markiearnold

Reply:
On second thought I don't think a capping component even needs to exist, just add these new options to buttons. We would still need the story to display it